.github/workflows/deploy.yml

```diff
@@ -3,6 +3,8 @@ name: build and deploy 2022.carpentrycon.org
 on:
   push:
     branches: main
+  schedule:
```

Comment:
are there any external data that feed into this site? I don't think so, so we don't need to build on schedule like we do for our other sites that need to be updated with the data coming from the feeds.